File: 250312/최대공약수 구하기/find-the-greatest-common-divisor.cpp
#include <iostream>

using namespace std;



void num(int n, int m){
    for(int i=m; i>0; i--){
        if(n%i==0 && m%i==0){
            cout << i;
            break;
        }
        else{
            continue;
        }
    }

}
    

int main() {
    int n, m;
    cin >> n >> m;

    // Please write your code here.
    num(n,m);
    
        

    return 0;
}


            

File: 250312/최대공약수 구하기/find-the-greatest-common-divisor.py
n, m = map(int, input().split())

# Please write your code here.
def num(n,m):
    for i in range(n-1, 1 ,-1):
            if n % i == 0 and m % i == 0:
                print(i)
                break
            else:
                continue
num(n,m)
            

